src/ceph_mgr.cc

```diff
@@ -52,6 +53,8 @@ int main(int argc, const char **argv)
     usage();
   }
 
+  pick_addresses(g_ceph_context, CEPH_PICK_ADDRESS_PUBLIC);
+
```

Comment:
I believe a space is added Intentionally between function calls.
It is required, serves a purpose?